src/c-writer.cc

```diff
@@ -417,6 +421,24 @@ class CWriter {
 
   bool IsImport(const std::string& name) const;
 
+  bool IsVarargsImportedFunction(const std::string& name) const {
+    return
+      !options_.features.sandbox_enabled() &&
+      varargs_imported_functions_.find(name) != varargs_imported_functions_.end();
```

Comment:
nit: contains?

Reply:
As discussed offline - let's switch to using "contains" later.